src/index.d.ts

```diff
@@ -60,6 +60,20 @@ declare module "lodash" {
      */
     consoleLogTime(label: string, fn: () => any): Promise<void>;
 
+    /**
+     * Part of `lodash-firecloud`.
+     *
+     * A Proxy to process.env that will throw if a variable is undefined.
+     */
+    env: { [key: string]: string };
+
+    /**
+     * Part of `lodash-firecloud`.
+     *
+     * Create Proxy to a process.env (or similar) object that will throw if a variable is undefined.
+     */
+    envProxy({ [key: string]: string }): { [key: string]: string };
```

Comment:
Why do we need a public `envProxy` function? Can you suggest possible scenarios?

Reply:
almost all js executables would have a "safer" runtime

example: https://github.com/tobiipro/atex-dashboard/pull/191/files#diff-c121422e89463500136d342b4b5fe60cR12